src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSinkUnitTest.java

```diff
@@ -18,8 +19,8 @@
 
 
 public class ReadsSparkSinkUnitTest extends BaseTest {
-    @DataProvider(name = "loadReads")
-    public Object[][] loadReads() {
+    @DataProvider(name = "loadReadsBAM")
+    public Object[][] loadReadsBAM() {
         String dir = "src/test/resources/org/broadinstitute/hellbender/tools/BQSR/";
```

Comment:
Check this after I push the next commit, just to make sure I'm using the right files.